mutiny-core/src/federation.rs

```diff
@@ -636,13 +695,7 @@ impl<S: MutinyStorage> FederationClient<S> {
     }
 
     pub async fn recover_backup(&self) -> Result<(), MutinyError> {
-        self.fedimint_client
-            .restore_from_backup(None)
-            .await
-            .map_err(|e| {
-                log_error!(self.logger, "Could not restore from backup: {}", e);
-                e
-            })?;
+        // todo this now a part of init
```

Comment:
What do you mean by this? Init should not be trying to call the restore functions.

Reply:
There's no longer the `restore_from_backup` function, instead you need to call it when building the client.

Reply:
Okay then it can be deleted then. Not used in mutiny web.